src/StreamsServiceProvider.php

```diff
@@ -677,42 +677,16 @@ protected function extendArr()
     /**
      * Extend the string utility.
      */
-    protected function extendStr()
+    protected function extendStr(): void
     {
-        Str::macro('humanize', function ($value, $separator = '_') {
-            return preg_replace('/[' . $separator . ']+/', ' ', strtolower(trim($value)));
```

Comment:
Here was an issue. If `$separator` be some regex char like `/` - this will lead to error.